src/components/FlightsSearchBar/FlightsSearchBar.js

```diff
@@ -1,17 +1,54 @@
 import RouteSearch from './RouteSearch/RouteSearch';
 import Passengers from './Passengers/Passengers';
 
+import useQueryString from '../../hooks/useQueryString';
+
 import styled from 'styled-components';
 import DatePick from './DatePick/DatePick';
 
 const FlightsSearchBar = () => {
+  const { searchParams, navigateToWithQueryString } = useQueryString();
+
+  function getDestructuredParamsByKeys(keys) {
+    return keys.reduce((acc, queryKey) => {
+      acc[queryKey] = searchParams.get(queryKey);
+      return acc;
+    }, {});
+  }
+
+  const addSearchedData = () => {
+    // 로컬스토리지에 저장
+    const paramsValues = getDestructuredParamsByKeys([
+      'departure_city',
+      'arrival_city',
+      'departure_date',
+      'arrival_date',
+      'adult',
+    ]);
+
+    let localData = localStorage.getItem('searchedTicket');
+    if (localData == null) localData = [];
```

Comment:
```suggestion
    const localData = localStorage.getItem('searchedTicket') ?? []
```
이렇게 하면 같은 의미일 것 같습니다
지금 방식은 let으로 선언한 변수를 변경해야 하고 null을 일치 연산자(===)로 비교해주고 있지 않아서 결과 예상이 어려워요